README.md

```diff
@@ -146,32 +148,51 @@ The provided extension method is `AddSignInWithAppleWithCompatibility`. It accep
 
 Sample code:
 ```csharp
-using AppleAuth.Editor;
+#if UNITY_IOS || UNITY_TVOS || UNITY_VISIONOS
+#define UNITY_XCODE_EXTENSIONS_AVAILABLE
+#endif
 
-public static class SignInWithApplePostprocessor
-{
-    [PostProcessBuild(1)]
-    public static void OnPostProcessBuild(BuildTarget target, string path)
-    {
-        if (target != BuildTarget.iOS)
-            return;
+using AppleAuth.Editor;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEngine;
+#if UNITY_XCODE_EXTENSIONS_AVAILABLE
+using UnityEditor.iOS.Xcode;
+#endif
 
-        var projectPath = PBXProject.GetPBXProjectPath(path);
-        
-        // Adds entitlement depending on the Unity version used
+namespace AppleAuthSample.Editor {
+  public static class SignInWithApplePostprocessor {
+    private const int CallOrder = 1;
+
+    [PostProcessBuild(CallOrder)]
+    public static void OnPostProcessBuild(BuildTarget target, string path) {
+      if (target == BuildTarget.iOS || target == BuildTarget.tvOS || target == BuildTarget.VisionOS) {
+#if UNITY_XCODE_EXTENSIONS_AVAILABLE
+        string projectPath = PBXProject.GetPBXProjectPath(path);
+        if (target == BuildTarget.VisionOS) {
+          // Unity's bug:
+          // After switch to VisionOS platform the projectPath is still "xx/Unity-iPhone.xcodeproj/project.pbxproj",
+          // while the expected path is "xx/Unity-VisionOS.xcodeproj/project.pbxproj",
+          // temporary fix:
+          projectPath = projectPath.Replace("Unity-iPhone.xcodeproj", "Unity-VisionOS.xcodeproj");
+        }
```

Comment:
The current adoption for visionOS is not that big.
I would rather have this code in a separate section, just before the macOS Plugin setup section and leave this documentation for iOS/tvOS as it was.

```
## Plugin setup (visionOS)

On visionOS, the setup is pretty much the same as the iOS/tvOS setup.
However, due to how Unity's visionOS project is exported, you need to adapt the Postprocess code to find the proper xcode project filename:

if (target == BuildTarget.VisionOS)
{
    projectPath = projectPath.Replace("Unity-iPhone.xcodeproj", "Unity-VisionOS.xcodeproj");
}
```

Reply:
done